database-mysqldb/index.js

```diff
@@ -0,0 +1,8 @@
+const Sequelize = require('sequelize');
+
+const db = new Sequelize('stocks', 'root', '', {
+  host: 'localhost',
+  dialect: 'mysql',
+});
+
+module.exports = db;
```

Comment:
@iamjaytong @wwong007 Good separation of models into their own files. Consider checking for errors on database connection.